### PR TITLE
 DBZ-2113 OpenShift tests for SQL Server Connector

### PR DESCRIPTION
**Pull request by jcechace**
https://issues.redhat.com/browse/DBZ-2113

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane, could you check this one out?

File: debezium-testing/debezium-testing-openshift/src/test/java/io/debezium/testing/openshift/resources/ConfigProperties.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
     public static final String OCP_PROJECT_DBZ = stringProperty("test.ocp.project.debezium");
     public static final String OCP_PROJECT_MYSQL = System.getProperty("test.ocp.project.mysql", "debezium-mysql");
     public static final String OCP_PROJECT_POSTGRESQL = System.getProperty("test.ocp.project.postgresql", "debezium-postgresql");
+    public static final String OCP_PROJECT_SQLSERVER = System.getProperty("test.ocp.project.sqlserber", "debezium-sqlserver");
```
Comment (jpechane) [inline debezium-testing/debezium-testing-openshift/src/test/java/io/debezium/testing/openshift/resources/ConfigProperties.java:26]: Should not be `test.ocp.project.sqlserver`?

File: debezium-testing/debezium-testing-openshift/src/test/java/io/debezium/testing/openshift/sqlserver/SqlServerConnectorIT.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+         awaitAssert(() -> assertRecordsContain(connectorName + ".dbo.customers", "tom@test.com"));
+     }
+
+     @Test
+     @Order(5)
+     public void shouldBeDown() throws SQLException, IOException {
+         kafkaConnectController.undeployConnector(connectorName);
+         String sql = "INSERT INTO customers (first_name, last_name, email) VALUES ('Jerry', 'Tester', 'jerry@test.com')";
+         dbController.executeStatement(DATABASE_SQLSERVER_DBZ_DBNAME, DATABASE_SQLSERVER_DBZ_USERNAME, DATABASE_SQLSERVER_DBZ_PASSWORD, sql);
+         awaitAssert(() ->assertRecordsCount(connectorName + ".dbo.customers", 5));
```
Comment (jpechane) [inline debezium-testing/debezium-testing-openshift/src/test/java/io/debezium/testing/openshift/sqlserver/SqlServerConnectorIT.java:130]: Missing space after `->`

**Comment by gunnarmorling** [comment]
@jcechace, can we wrap up this one?

**Comment by jcechace** [comment]
@gunnarmorling  Will need some refactoring to play nicely with Mongo. In order to avoid cherry-picks and rebases I'm closing this sone and will send a new PR on Monday containing both -- Mongo and SqlServer together. Is that ok?

**Comment by gunnarmorling** [comment]
Do as you prefer. In generally, multiple smaller PRs are easier to handle.

>

**Pull request closed without merging**